patches/components-permissions-android-permission_dialog_delegate.cc.patch

```diff
@@ -1,8 +1,16 @@
 diff --git a/components/permissions/android/permission_dialog_delegate.cc b/components/permissions/android/permission_dialog_delegate.cc
-index c76378df51a497a9cc53015893e33fa2ae19f1d4..214402321ed8b9f57963832578e636fafee51c9b 100644
+index c76378df51a497a9cc53015893e33fa2ae19f1d4..523167192972d39790b0923c15af851904ec1548 100644
 --- a/components/permissions/android/permission_dialog_delegate.cc
 +++ b/components/permissions/android/permission_dialog_delegate.cc
-@@ -95,12 +95,14 @@ PermissionDialogDelegate* PermissionDialogDelegate::CreateForTesting(
+@@ -59,6 +59,7 @@ void PermissionDialogJavaDelegate::CreateDialog() {
+   // The controller takes over lifetime management; when the Java delegate is no
+   // longer needed it will in turn free the native delegate
+   // (PermissionDialogDelegate).
++  BRAVE_PERMISSION_DIALOG_DELEGATE_ADD_WALLET_PARAMS
```

Comment:
you can override `Java_PermissionDialogController_createDialog` to avoid adding this patch

Reply:
I cannot, as I need a class member `permission_prompt_`

Reply:
that code is removed and we have a permission dialog in a browser process now